### PR TITLE
[stable10] Bump latest symfony and sabre/vobject point versions

### PR DESCRIPTION
**Pull request by phil-davis**
Backport #29977

### CODE REVIEW

**Comment by phil-davis** [comment]
Note: this also bumps minor versions of other stuff in ``composer.lock`` up to similar versions that are in master. That is because the only reasonable way to resolve the backport conflicts for ``composer.lock`` is to run ``composer update`` again in ``stable10`` to make a nice new ``composer.lock`` - and of course that bumps everything it can find to the latest compatible versions.

File: composer.lock [hunk 1]
```diff
@@ -2875,42 +2889,43 @@
...
             ],
-            "time": "2017-01-28T02:37:08+00:00"
+            "time": "2018-01-04T15:09:34+00:00"
         }
     ],
     "packages-dev": [
         {
             "name": "behat/behat",
-            "version": "v3.3.0",
+            "version": "v3.4.3",
```
Comment (DeepDiver1975) [inline composer.lock]: I prefer to only update libs which are related to this change.

Comment (phil-davis) [reply composer.lock]: To resolve conflicts in trying to backport ``composer.lock`` it is really necessary to run ``composer update`` and let it regenerate ``composer.lock`` again. That file is not the sort of thing to hand-edit.
So this here is the result. 
What to do?

Comment (DeepDiver1975) [reply composer.lock]: to update the lock file try running
```sh
composer update --lock
```

Comment (PVince81) [reply composer.lock]: Try `make update-composer` instead, it should regenerate the lock file. Not sure if that's the same...

Comment (phil-davis) [reply composer.lock]: I used ``make update-composer`` this time while doing a rebase to resolve conflicts. That produces a similar result - plenty of bumps of versions like this.

**Comment by phil-davis** [comment]
See #30266 for another try at this.

**Comment by patrickjahns** [comment]
@phil-davis 
To update just one library - it is straight forward `composer update <library> ` - in this case `composer update sabre/vobject `

**Comment by phil-davis** [comment]
@patrickjahns #30266 has a better attempt at this.
The other version bumps that happen here if you do ``composer update`` are things that have already happened in ``master`` and need/should/could be bumped in ``stable10`` also.

**Comment by phil-davis** [comment]
Done in #30266

**Pull request closed without merging**